### PR TITLE
Add human-readable part for segwit addresses to chaincfg.Params

### PR DESCRIPTION
**Pull request by halseth**
To be used together with https://github.com/btcsuite/btcutil/pull/100

### CODE REVIEW

**Comment by stevenroose** [comment]
I would prefer a more declarative name than `Hrp`. First of all, abbreviations are supposed to be all upper case, so it'd be `HRP`, but I'm more in favor of something like `Bech32SegwitHRP`.

**Comment by halseth** [comment]
Good point! I wonder if maybe `SegwitBech32HRP` would be even better, to emphasize that this is for segwit addresses? (and since `hrp` is Bech32 terminology I think it makes sense to have it next to `Bech32` 😄 )

**Comment by stevenroose** [comment]
Well, if there will be more prefixes in the future, it's nice to have them looking similar, that's why those are often named hierarchical. Bech32HRPSegWit is hierarchical

**Comment by halseth** [comment]
Agreed, done.

**Comment by stevenroose** [comment]
(You can squash both commits into the original one like this:
```
git reset HEAD~1
git commit --amend
git push -f
```
So we don't need two commits for this small a change :) )

**Comment by halseth** [comment]
Done 👍

File: chaincfg/params.go [hunk 1]
```diff
@@ -476,6 +492,10 @@ var SimNetParams = Params{
 	// Mempool parameters
 	RelayNonStdTxs: true,
 
+	// Human-readable part for Bech32 encoded segwit addresses, as defined in
+	// BIP 173.
+	Bech32HRPSegwit: "tb", // always tb for test net
```
Comment (davecgh) [inline chaincfg/params.go]: Can we make this "sb" (and adjust the comments accordingly)?  We don't want simnet to be able to use testnet addresses.  Hence why the p2pkh addrs start with S (just below).

Comment (halseth) [inline chaincfg/params.go]: I guess you are referring to `SimNetParams`? Just pushed the requested change there :)

Comment (davecgh) [inline chaincfg/params.go]: I was, yep.  I commented on that specific line.  Thanks for the update!

**Pull request merged**